### PR TITLE
implementing services for item

### PR DESCRIPTION
**Pull request by juanDuqueS**
Implementing services for item

What: trying to implement the services for the endpoints to connect with the database

Why: to have a functional API with its crud

### CODE REVIEW

**Review by alejandroVaron — APPROVED** [review]
nice!

**Comment by AngelicaAgudelo** [comment]
Missing description -_-

**Review by prometteo14 — APPROVED** [review]

File: back-end-yaz/src/server/controllers/ItemController.js [hunk 1]
```diff
@@ -7,7 +7,12 @@ class ItemController {
 
     static async getAllItems(req, res) {
         try {
-            util.setSuccess(200, `All items returned!`);
+            const allItems = await ItemService.getAllItems();
+            if(allItems.length > 0){
+                util.setSuccess(200, 'Items returned', allItems)
+            } else {
```
Comment (AngelicaAgudelo) [inline back-end-yaz/src/server/controllers/ItemController.js:13]: Are you sure about the error code?

Comment (juanDuqueS) [reply back-end-yaz/src/server/controllers/ItemController.js:13]: nope, it has to be reviewed later :P

Comment (AngelicaAgudelo) [reply back-end-yaz/src/server/controllers/ItemController.js:13]: Please create a ticket to the next sprint to review this

**Review by AngelicaAgudelo — APPROVED** [review]
Ok.

**Comment by juanDuqueS** [comment]
> Missing description -_-

already changed, sorry

**Pull request merged**